samples/multi-tenant/ravendb/Core_7/Receiver/Program.cs

```diff
@@ -51,7 +48,7 @@ static async Task Main()
 
             #region DisableOutboxCleanup
 
-            endpointConfiguration.SetFrequencyToRunDeduplicationDataCleanup(Timeout.InfiniteTimeSpan);
+            outbox.SetFrequencyToRunDeduplicationDataCleanup(Timeout.InfiniteTimeSpan);
```

Comment:
Given that the sample uses the latest version, this shouldn't be required. Correct?

Reply:
yeah I missed that

Reply:
Fixed